src/Api/Domain.php

```diff
@@ -299,4 +303,80 @@ public function verify(string $domain)
 
         return $this->hydrateResponse($response, VerifyResponse::class);
     }
+
+    /**
+     * Returns a domain tracking settings.
+     *
+     * @param string $domain name of the domain
+     *
+     * @return TrackingResponse|array|ResponseInterface
+     */
+    public function tracking(string $domain)
+    {
+        Assert::stringNotEmpty($domain);
+
+        $response = $this->httpGet(sprintf('/v3/domains/%s/tracking', $domain));
+
+        return $this->hydrateResponse($response, TrackingResponse::class);
+    }
+
+    /**
+     * Updates a domain click tracking settings.
+     *
+     * @param string $domain name of the domain
+     *
+     * @param bool $active
+     * @return UpdateClickTrackingResponse|array|ResponseInterface
+     */
+    public function updateClickTracking(string $domain, bool $active)
+    {
+        $params = [
+            'active' => $active ? 'true' : 'false',
+        ];
+
+        $response = $this->httpPut(sprintf('/v3/domains/%s/tracking/click', $domain), $params);
+
+        return $this->hydrateResponse($response, UpdateClickTrackingResponse::class);
+    }
+
+    /**
+     * Updates a domain open tracking settings.
+     *
+     * @param string $domain name of the domain
+     * @param bool $active
+     *
+     * @return UpdateOpenTrackingResponse|array|ResponseInterface
+     */
+    public function updateOpenTracking(string $domain, bool $active)
+    {
+        $params = [
+            'active' => $active ? 'true' : 'false',
```

Comment:
According to the Mailgun Documentation, the value should be `yes` or `no` instead of `true` or `false`.

Can you fix this implementation, please, and make sure the automated tests are checking the expected values?

Please don't forget adding the expected `Asset::...` before setting the parameters. It would be great if you also add additional unit tests to validate the exceptions thrown by the `Asset` methods.

https://documentation.mailgun.com/en/latest/api-domains.html#domains

![image](https://user-images.githubusercontent.com/7809429/95641888-e3535f80-0a9c-11eb-8b82-ebd5ef4766ff.png)